evalml/data_checks/high_variance_cv_data_check.py

```diff
@@ -37,7 +37,10 @@ def validate(self, pipeline_name, cv_scores):
             cv_scores = pd.Series(cv_scores)
 
         messages = []
-        high_variance_cv = abs(cv_scores.std() / cv_scores.mean()) > self.threshold
+        if cv_scores.mean() == 0 and cv_scores.std() == 0:
```

Comment:
Would this work if we only check if the mean is equal to 0?

Reply:
mm yeah I think so. Since these are scores, if the mean is 0, the std is also 0. Thanks for pointing that out!